crates/holochain/src/conductor/manager/mod.rs

```diff
@@ -161,7 +217,61 @@ mod test {
         if let Err(_) = send_task_handle.send(handle).await {
             panic!("Failed to send the handle");
         }
-        main_handle.await??;
+        main_handle.await???;
         Ok(())
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    #[should_panic]
+    async fn unrecoverable_error() {
```

Comment:
I'm also confused about the name, as to me it looks like we can recover from this error :thinking: I'm probably misunderstanding the code and am happy to be corrected on this one

Reply:
We don't want to recover from these errors right now. Some are actually unrecoverable like malformed database and others are things we will try to recover from in the future but for now it's easier to debug by failing early.